.github/workflows/deploy.yml

```diff
@@ -0,0 +1,141 @@
+name: Deploy to Demo and Test
+on:
+  push:
+    branches:
+      - master
+      - develop
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v1
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+
+      # Build for demo
+      - name: Build and publish demo docker image
+        if: github.ref == 'refs/heads/master'
+        uses: elgohr/Publish-Docker-Github-Action@master
+        with:
+          name: demo-storefrontcloud-io/vue-storefront-api:${{ github.sha }}
+          registry: registry.storefrontcloud.io
+          username: ${{ secrets.CLOUD_USERNAME }}
+          password: ${{ secrets.CLOUD_PASSWORD }}
+          dockerfile: dev/docker/Dockerfile
+          buildoptions: "--compress"
+
+      # Build for test
+      - name: Build and publish test docker image
+        if: github.ref == 'refs/heads/develop'
+        uses: elgohr/Publish-Docker-Github-Action@master
+        with:
+          name: test-storefrontcloud-io/vue-storefront-api:${{ github.sha }}
+          registry: registry.storefrontcloud.io
+          username: ${{ secrets.CLOUD_USERNAME }}
+          password: ${{ secrets.CLOUD_PASSWORD }}
+          dockerfile: dev/docker/Dockerfile
+          buildoptions: "--compress"
+
+  deploy-demo:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - uses: chrnorm/deployment-action@releases/v1
+        if: github.ref == 'refs/heads/master'
+        name: Create GitHub deployment for demo
+        id: deployment
+        with:
+          token: "${{ github.token }}"
+          target_url: https://demo.europe-west1.gcp.storefrontcloud.io/api/
+          environment: production
+          initial_status: in_progress
+      # Deploy on demo
+      - name: Deploy on demo.europe-west1.gcp.storefrontcloud.io
+        if: github.ref == 'refs/heads/master'
+        run: |
+          if curl -s -H 'X-User-Id: ${{ secrets.DEPLOYMENT_USER }}' -H 'X-Api-Key: ${{ secrets.DEPLOYMENT_PASSWORD }}' -H 'Content-Type: application/json' -X POST -d '
```

Comment:
Now, you should use CLOUD_USERNAME and CLOUD_PASSWORD instead of DEPLOYMENT_USER and DEPLOYMENT_PASSWORD